perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -210,10 +206,11 @@ private static PerunPrincipal setupPerunPrincipal(HttpServletRequest req, Deseri
 
 		// If OIDC_CLAIM_sub header is present, it means user authenticated via OAuth2 with MITRE.
 		else if (isNotEmpty(req.getHeader(OIDC_CLAIM_SUB))) {
-			extSourceName = req.getHeader(OIDC_CLAIM_EXTSOURCE_NAME);
-			extSourceType = req.getHeader(OIDC_CLAIM_EXTSOURCE_TYPE);
-			extSourceLoaString = req.getHeader(OIDC_CLAIM_EXTSOURCE_LOA);
-			extLogin = remoteUser;
+			//this is configurable, as the OIDC server has the sub claim also configurable
+			extSourceName = config.getOidcExtsourceName();
+			extSourceType = config.getOidcExtsourceType();
+			extSourceLoaString = "N/A";
```

Comment:
On many places we expect LoA to be a number (even when it's retrieved as a string). If unsure about trust of source identity I suggest to use "0".